OpenRA.Mods.Common/UtilityCommands/CheckYaml.cs

```diff
@@ -62,6 +63,13 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 
 					// Run all rule checks on the default mod rules.
 					CheckRules(modData, modData.DefaultRules);
+					foreach (var tileset in modData.DefaultTerrainInfo.Keys)
```

Comment:
Genuine question: Will these always hold?

Rules being `DefaultRules`
TerrainInfo being `DefaultTerrainInfo`
FileSystem being `DefaultFileSystem`

It seems a bit contrived, as though we are implying extensibility when there isn't any. Maybe a `TO DO: Provide support for multiple / non-standard TerrainInfos` or similar is in order? Otherwise you may as well remove the 'Default' keyword if no alternatives are planned.

Reply:
TerrainInfo can't (yet...?) be modified by maps. I agree, but this request has nothing to do with this PR!

Reply:
Well it does directly relate to code you are adding, so I think adding a 'TO DO:' would not be out of line, but your call.